aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/data/AmplifyCredential.kt

```diff
@@ -134,9 +141,9 @@ internal data class AWSCredentials(
     }
 
     override fun toString(): String = "AWSCredentials(" +
-        "accessKeyId = ${accessKeyId?.substring(0..4)}***, " +
-        "secretAccessKey = ${secretAccessKey?.substring(0..4)}***, " +
-        "sessionToken = ${sessionToken?.substring(0..4)}***, " +
+        "accessKeyId = ${accessKeyId.mask()}, " +
```

Comment:
This really probably doesn't matter since its a masked value but this method includes 1 less character.